doc/changes/whats_new.rst

```diff
@@ -3,7 +3,10 @@
 
 What's new
 ==========
+NEW
+---
 
+- New radiological view option for cross-sectional MRI plotting 
```

Comment:
Could you put this in `latest.rst` following the same format as the entries already there?